### PR TITLE
Quiz page

### PR DESCRIPTION
**Pull request by athar05**
This PR contains:

Context for Results, score & user
A reducer for results
A fully functional quiz card
Mobile responsive UI

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -156,6 +172,107 @@ a {
...
+  justify-content: space-around;
+  margin: 10px;
+}
+
+.single-option {
+  width: 46%;
+  height: 50px;
+  padding: 15px 20px;
+  margin: 10px;
+  /* font-size: 1rem; */
```
Comment (rahul31agrawal) [inline src/App.css]: remove comented code

File: src/App.css [hunk 2]
```diff
@@ -156,6 +172,107 @@ a {
...
+  background-color: var(--gray);
+  /* box-shadow: 0 0 2px black; */
```
Comment (rahul31agrawal) [inline src/App.css]: same as above

**Review by rahul31agrawal — APPROVED** [review]
Hi Aakhib,

1) The code structure looks good
2) UI looks neat
3) the result page is not showing the result, it only says "this is result".
4) Please check below screenshot , that needs to be corrected
bug

File: src/App.css [hunk 3]
```diff
@@ -156,6 +172,107 @@ a {
...
+}
+
+.quiz-header {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.quiz-details{
+  /* border: 1px solid black; */
```
Comment (chanchal16) [inline src/App.css]: remove the commented code

File: src/components/Questions-Body/QuestionsBody.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
+import React, { useState, useContext } from "react";
+import { Button } from "@mui/material";
+import { ScoreContext } from "../../context/ScoreContext.js";
+import { Link, useNavigate } from "react-router-dom";
+import { ResultContext } from "../../context/ResultContext.js";
```
Comment (chanchal16) [inline src/components/Questions-Body/QuestionsBody.js:5]: Suggestion: You can use barrel exports for contexts

**Review by chanchal16 — APPROVED** [review]

File: src/components/Questions-Body/QuestionsBody.js (hunk 4 above)
Comment (athar05) [reply src/components/Questions-Body/QuestionsBody.js:5]: Thank you for the sugession

File: src/App.css (hunk 3 above)
Comment (athar05) [reply src/App.css]: Done

File: src/App.css (hunk 2 above)
Comment (athar05) [reply src/App.css]: Done

File: src/App.css (hunk 1 above)
Comment (athar05) [reply src/App.css]: Done

**Pull request merged**